packages/financial-templates-lib/src/price-feed/BasketSpreadPriceFeed.js

```diff
@@ -134,7 +135,55 @@ class BasketSpreadPriceFeed extends PriceFeedInterface {
       return this._getSpreadFromBasketPrices(experimentalPrices, baselinePrices, denominatorPrice);
     }
   }
+  // This searches for closest time in a list of [time,price] data. Based on code in affiliates models/prices.
+  closestTime(list) {
```

Comment:
You might want to add a comment explaining the expected shape of `list`

Reply:
there is a comment above but i think its not technically correct. i will update it to make it more clear